### ISSUE TITLE
Left/Right "Hints" giving away the answer

### ISSUE THREAD

**Issue by membranepotential** [elixirkoans/elixir-koans#195]
I love the koans and the app =)

However, I believe showing left/right values when an assertion fails gives away to much of the answer. For example:

```
The ~s sigil is a different way of expressing string literals
Assertion failed in lib/koans/11_sigils.ex:7
~s"This is a string" == "Hello"

left:  "This is a string"
right: "Hello"
```

I perceive koans as problems that come without any hints and can only be solved by thinking about them for days in a dark chamber isolated from civilization. Giving those hints makes it too easy sometimes.

Thank you for the good work!
Cheers =)

**Comment by krlsdu** [comment]
Good point! @membranepotential , maybe not necessarily a phrase, but a link to respective elixir documentation.
The thread of this approach it is maintains the link consistent because the elixir documentation can change, and check this on the test is really hard.
